ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -412,4 +408,14 @@ private void logReceivedBuilderBid(final BuilderBid builderBid) {
         payloadHeader.getGasLimit(),
         payloadHeader.getGasUsed());
   }
+
+  private void logLocalPayloadWin(
+      final SignedBuilderBid signedBuilderBid, final UInt256 localPayloadValue) {
+    LOG.info(
+        "The local execution payload value ({} Gwei) was awarded the block over the builder payload ({} Gwei), "
```

Comment:
Since we are changing it, the sentence doesn't sound very good.
something like "the local execution payload value () has been chosen over the the builder bid value ()."